bbn-test-5/babylon-node/README.md

```diff
@@ -0,0 +1,269 @@
+should never be used in production.
+- `file`: Stores encrypted keys on disk, which is a more secure option than test but 
+less secure than using the OS keyring.
+- `os`: Uses the operating system's native keyring, providing the highest level of 
+security by relying on OS-managed encryption and access controls.
+
+The execution result displays the address of the newly generated key and its 
+public key. Following is a sample output for the command:
+
+```shell
+- address: bbn1kvajzzn6gtfn2x6ujfvd6q54etzxnqg7pkylk9
+  name: <name>
+  pubkey: '{"@type":"/cosmos.crypto.secp256k1.PubKey",
+           key: "Ayau+8f945c1iQp9tfTVaCT5lzhD8n4MRkZNqpoL6Tpo"}'
+  type: local
+```
+
+Make sure to securely store this information, particularly your 
+address and private key details. Losing access to these credentials 
+would mean losing access to your account and any funds associated with it.
+
+## Sync Node
+
+We are now ready to sync the node.
+
+```shell
+babylond start --chain-id bbn-test-5 --home <path> --x-crisis-skip-assert-invariants
+```
+
+Lets go through the flags of the above command:
+
+- `start`: This is the command to start the Babylon node.
+- `--chain-id`: Specifies the ID of the blockchain network you're connecting to.
+- `--home`: Sets the directory for the node's data and configuration files and 
+is dependant on where the files were generated for you from the initialization (e.g. `--home ./nodeDir`)
+
+### Options for Syncing
+
+You have two options for syncing your node:
+
+1. **Sync through Network Peers**
+   - Use the seed node configuration mentioned earlier in `config.toml`
+   - Additional peers can be added under the `persistent_peers` setting
+   <!-- Add peer list when available -->
+
+2. **Sync from Snapshot**
+   - For faster syncing, you can use a snapshot instead of syncing from genesis
+   - Snapshots are periodic backups of the chain state
+   - Find available snapshots at: <!-- Add link when available -->
+   
+>Note: Always verify snapshot sources and checksums before using them to ensure security.
+
+## Get Funds
+
+To interact with the Babylon network, you'll need some BBN tokens to:
+1. Pay for transaction fees (gas)
+2. Send transactions
+3. Participate in network activities
+
+You can obtain testnet tokens through two methods:
+
+1. Request funds from the Babylon Testnet Faucet 
+[here](#tbd) 
+<!-- add link to faucet -->
+2. Join our Discord server and visit the #faucet channel: 
+[Discord Server](https://discord.com/channels/1046686458070700112/1075371070493831259)
+
+Note: These are testnet tokens with no real value, used only for testing 
+and development purposes.
+
+## Monitoring Your Node
+
+Your Babylon node exposes metrics through two Prometheus endpoints:
+- Port 1317: API metrics
+- Port 26660: Tendermint metrics
+
+To enable metric collection, modify your `app.toml`:
+```
+[telemetry]
+enabled = true
+prometheus-retention-time = 60
```

Comment:
```suggestion
prometheus-retention-time = 600
```
This will delete Prom metrics after 60s. To make up for scrape delays, suggest making it 10 min.